scripts/pi-hole/js/groups-adlists.js

```diff
@@ -117,24 +117,32 @@ function initTable() {
     order: [[0, "asc"]],
     columns: [
       { data: "id", visible: false },
-      { data: "status", searchable: false, class: "details-control" },
-      { data: "address" },
+      { data: null, visible: true, orderable: false, width: "15px" },
+      { data: "status", searchable: false, orderable: false, class: "details-control" },
```

Comment:
Why did you prevent ordering here?
```suggestion
      { data: "status", searchable: false, class: "details-control" },
```

Reply:
![sorting-padding](https://user-images.githubusercontent.com/1385443/166113064-9f26e407-3e5b-49b1-81b8-3342caf8f8fc.gif)

I was testing the column without the sorting and forgot to re-enabled it.